README.md

```diff
@@ -214,31 +212,32 @@ service studentData on studentServiceListener {
         var result = caller->respond(response);
         if (result is error) {
             // Log the error for the service maintainers.
-            log:printError("Error responding to the client", err = result);
+            log:printError("Error sending response", err = result);
         }
-
     }
 
+    // Resource configuration for viewing all the student's details in the system.
     @http:ResourceConfig {
         methods: ["POST"],
         path: "/viewAll"
     }
-    // View students resource is to get all the student's details and send to the requested user.
+    // View students resource is to get all the students details and send to the requested user.
```

Comment:
Resource configuration to get details of all the students and send to the requested user.